workshops/templates/workshops/all_memberships.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_nav_fluid_sidebar.html" %}
+
+{% load pagination %}
+
+{% block content %}
+    {% if perms.workshops.add_membership %}
+    <p><a href="{% url 'membership_add' %}" class="btn btn-success">New membership</a></p>
+    {% else %}
+    <p><a href="{% url 'membership_add' %}" class="btn btn-success disabled">New membership</a></p>
+    {% endif %}
+{% if all_memberships %}
+    <table class="table table-striped">
+        <tr>
+            <th>organization</th>
+            <th>kind</th>
+            <th>start date</th>
+            <th>end date</th>
+            <th>contribution</th>
+        </tr>
+    {% for membership in all_memberships %}
+        <tr>
+            <td><a href="{{ membership.get_absolute_url }}">{{ membership.organization.fullname }}</a></td>
```

Comment:
Isn't this confusing to add a hyperlink to the `membership_details` view here? I would suggest adding view and edit glyphicons like the ones found in `all_tasks` view.